Add DagBundle information to Serialized DagVersion table

ephraimbuddy opened the pull request:
This will help us keep track of the different versions that we see in DagBundle since DagModel would always have the latest bundle version.

jedcunningham commented on airflow/models/serialized_dag.py:
```diff
@@ -201,8 +201,13 @@ def write_dag(
         if serialized_dag_hash is not None and serialized_dag_hash == new_serialized_dag.dag_hash:
             log.debug("Serialized DAG (%s) is unchanged. Skipping writing to DB", dag.dag_id)
             return False
+        dag_model = session.execute(
```
It's probably better to pass the bundle_name and bundle_version down from bulk_sync_to_db to here instead of querying it.

VVRChilukoori replied:
Agree with @jedcunningham 's comments

ephraimbuddy committed:
Add DagBundle information to Serialized DagVersion table

This will help us keep track of the different versions that we see
in DagBundle since DagModel would always have the latest bundle version.

ephraimbuddy committed:
fixup! Add DagBundle information to Serialized DagVersion table

ephraimbuddy committed:
Fix test
